### PR TITLE
Add python 3.10 docker, update readme

### PR DESCRIPTION
**Pull request by tammojan**

### CODE REVIEW

**Comment by tammojan** [comment]
For reference, I fixed the mac build by updating the hardcoded date in the WSRT_Measures file in the repository https://github.com/casacore/homebrew-tap

File: README.rst [hunk 1]
```diff
@@ -93,4 +80,5 @@ Credits
 
 * `Ger van Diepen <gervandiepen@gmail.com>`_
 * `Malte Marquarding <Malte.Marquarding@gmail.com>`_
+* `Tammo Jan Dijkema <dijkema@astron.nl>`_
```
Comment (mnijhuis-tos) [inline README.rst:83]: Why didn't you add yourself at the end?

**Review by mnijhuis-tos — APPROVED** [review]
I only have one small question.

File: README.rst (hunk 1 above)
Comment (tammojan) [reply README.rst]: Didn't think about the order, indeed at the end is more correct.

**Pull request merged**